app/models/trail.rb

```diff
@@ -1,4 +1,16 @@
 class Trail < ActiveRecord::Base
   has_many :activities
   has_many :species
+
+  scope :filter_by_state, -> (state) {
+    where("lower(state) LIKE ?", "%#{state.downcase}%")
+  }
+
+  scope :filter_by_lat, -> (lat) {
+    where("lat BETWEEN ? AND ?", lat - 1, lat + 1)
+  }
+
+  scope :filter_by_long, -> (long) {
```

Comment:
Although I also personally prefer the `long` abbreviation, perhaps change to `:filter_by_lng` since we are using the `lng` abbreviation rather than `long` elsewhere in the project.